llvm/lib/Target/RISCV/RISCVSelectionDAGInfo.h

```diff
@@ -11,15 +11,48 @@
 
 #include "llvm/CodeGen/SelectionDAGTargetInfo.h"
 
+#define GET_SDNODE_ENUM
+#include "RISCVGenSDNodeInfo.inc"
+
 namespace llvm {
 
-class RISCVSelectionDAGInfo : public SelectionDAGTargetInfo {
+namespace RISCVISD {
+// RISCVISD Node TSFlags
+enum : unsigned {
```

Comment:
uint32_t

Reply:
One better: introduced a `using` for this in a target-independent header, so that if tsflags is extended it only needs to be updated in one place... hopefully.